app/src/main/java/com/techcourse/controller/LogoutController.java

```diff
@@ -1,15 +1,22 @@
 package com.techcourse.controller;
 
+import context.org.springframework.stereotype.Controller;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import webmvc.org.springframework.web.servlet.mvc.asis.Controller;
+import web.org.springframework.web.bind.annotation.RequestMapping;
+import web.org.springframework.web.bind.annotation.RequestMethod;
+import webmvc.org.springframework.web.servlet.ModelAndView;
+import webmvc.org.springframework.web.servlet.view.JspView;
 
-public class LogoutController implements Controller {
+@Controller
+public class LogoutController {
 
-    @Override
-    public String execute(final HttpServletRequest req, final HttpServletResponse res) throws Exception {
+    @RequestMapping(value = "/logout", method = RequestMethod.GET)
+    public ModelAndView logout(final HttpServletRequest req, final HttpServletResponse res) throws Exception {
```

Comment:
소소하기도 하고 해당 미션에서는 굳이 적용하지 않아도 되지만 res같이 사용하지 않는 파라미터의 경우 메서드 시그니처만으로 어떤 파라미터가 사용되는지 확인할 수 있도록 `ignored`를 붙여주시는 것도 나쁘지는 않을 것 같습니다!

Reply:
```suggestion
    public ModelAndView logout(final HttpServletRequest ignored, final HttpServletResponse ignored) throws Exception {
```
<br/>

`ignored`가 이거 말하는게 맞을까요? 1단계인가 2단계에서 썼던 것 같은데 가물가물하네요.. 

이거 안쓰게 된 이유가 어차피 사람들은 IDE를 통해 확인할거고, IDE에서는 `request`, `response`가 회색으로 보이기 때문에 문제 없을거라고 생각했어요

근데 이것도 적다보니 의문이 드는건데 인텔리제이에서는 확인할 수 있지만, 이클립스에서는 잘 모르겠고, vim으로 코딩하는 곳도 있다던데 여기서는 이걸 확인하지 못할 것 같네요 😅

컨트롤러 부분을 전체적으로 확인하면서 수정했습니다!

Reply:
넵 맞습니다! 저도 기존에는 IDE가 보여주니까 상관없다고 생각했는데 협업을 진행하면서 깃허브나 기타 등등 환경에서 IDE 없이 보는 상황이 많아져서 이런 식으로 명시를 해주니 좀 읽기가 편하더라고요 

그리고 사실..리뷰할 내용이 너무 없어서 좀 억지로 끼워넣은 느낌이기는 한데 반영해주셔서 감사합니다

Reply:
아니에요 저도 오랜만에 사용해봐서 좋았어요 ㅋㅋ
레벨 1, 2에서 한게 많은 것 같은데 그만큼 잊혀진 것도 많았네요